docs/Features.md

```diff
@@ -8,4 +8,5 @@
 - [RelaxNG features](Features/RelaxNGFeatures.md#relaxng-features)
 - [XInclude features](Features/XIncludeFeatures.md#xinclude-features)
 - [XML Catalog features](Features/XMLCatalogFeatures.md#xml-catalog-features)
-- [XML References features](Features/XMLReferencesFeatures.md#xml-references-features)
\ No newline at end of file
+- [XML References features](Features/XMLReferencesFeatures.md#xml-references-features)
+- [XML Colors features](Features/XMLReferencesFeatures.md#xml-colors-features)
```

Comment:
This link should point to Features/XMLColorsFeatures.md#xml-colors-features

Reply:
Thanks, fixed.